src/client/pages/Offer/Introduction/Sidebar/index.tsx

```diff
@@ -248,7 +259,10 @@ export const Sidebar: React.FC<SidebarProps> = ({
                     to={`/${localePath}/new-member/checkout/details/${quoteCartId}`}
                     foreground={colorsV3.gray900}
                     background={colorsV3.purple500}
-                    disabled={isLoadingQuoteCart}
+                    disabled={isLoading}
+                    onClick={(e) => {
+                      if (isLoading) e.preventDefault()
+                    }}
```

Comment:
Curious where this comes from. What does it mean that we want to prevent default behavior if loading?

Reply:
Since that's actually a link button, and can be clicked even if disabled={true}, it needed this to prevent from switching to the next page. 😅

Reply:
🤔 I thought that by default a link wouldn’t be able to be clicked when it's disabled as it happens with `<button>`s but it seems not to be the case. After some quick search, it seems people use  CSS `pointer-events: none` to implement disabled links, so there's this option as well in case you're up to an alternative solution.